cypress/integration/executions_spec.js

```diff
@@ -69,5 +69,59 @@ describe('Dashboard Executions Page', () => {
         });
       });
     });
+
+    it('should show a single execution', () => {
+      cy.contains('nav li a', 'Executions').as('executions');
+      cy.get('@executions').should('have.attr', 'href', '#/executions');
+      cy.get('@executions').click();
+
+      cy.url().should('include', 'executions');
+      cy.contains('.heading--xlarge', 'Executions');
+      const executionName = '50eaad71-bba8-4376-83d7-bb9cc1309b92';
+      const executionArn = 'arn:aws:states:us-east-1:596205514787:execution:TestSourceIntegrationIngestGranuleStateMachine-MOyI0myKEXzf:50eaad71-bba8-4376-83d7-bb9cc1309b92';
+      const stateMachine = 'arn:aws:states:us-east-1:596205514787:stateMachine:TestSourceIntegrationIngestGranuleStateMachine-MOyI0myKEXzf';
+      cy.get('table tbody tr td[class=table__main-asset]').within(() => {
+        cy.get(`a[title=${executionName}]`).click({force: true});
+      });
+
+      cy.contains('.heading--large', 'Execution');
+      cy.contains('.heading--medium', 'Visual workflow');
+
+      cy.get('.table--wrapper')
+        .within(() => {
+          let i;
+          for (i = 26; i < 32; i++) {
+            let idMatch = `"id": ${i},`;
+            let previousIdMatch = `"previousEventId": ${i - 1}`;
+
+            cy.get(`tr[data-value=${i}]`).children('td').as('items');
+            cy.get('@items').eq(0).should('have.text', i.toString());
+            cy.get('@items').eq(2).invoke('text').should('match', /\d{2}:\d{2}:\d{2} \d{2}\/\d{2}\/\d{2}$/);
+            cy.get('@items').eq(3).contains('More Details').click();
+            cy.get('@items').eq(3).contains(idMatch).contains(previousIdMatch);
+            cy.get('@items').eq(3).contains('Less Details').click();
+          }
+        });
+
+      cy.get('.status--process')
+        .within(() => {
+          cy.contains('Execution Status:').next().should('have.text', 'Succeeded');
+          cy.contains('Execution Arn:').next().should('have.text', executionArn);
+          cy.contains('State Machine Arn:').next().should('have.text', stateMachine);
+          cy.contains('Started:').next().should('have.text', '20:05:10 11/12/18');
```

Comment:
When I run this locally, I get a failure of `expected '<dd>' to have text '20:05:10 11/12/18', but the text was '15:05:10 11/12/18'`

Reply:
I saw this too.  I think the baseline code doesn't convert the timestamp string to another format correctly.

Reply:
I think this is happening because `moment` defaults to the local timezone, which for all of us is east coast time. This command returns `15:05:10 11/12/18` for me:

`moment('2018-11-12T20:05:10.401Z').format('kk:mm:ss MM/DD/YY');`

To ensure that the test compares against the correct value for its timezone, we should import the `fullDate` function in `utils/format` that generates the timestamp for the dashboard and use it for the comparison in the test